codechecker_lib/context_base.py

```diff
@@ -60,6 +64,16 @@ def set_env(self, env_vars):
         if codechecker_workspace:
             self._codechecker_workspace = codechecker_workspace
 
+        try:
+            with open(self.checkers_severity_map_file) as severity_file:
+                self._severity_map = json.load(severity_file)
+        except IOError:
+            LOG.warning('No such file: {}'.format(
+                self.checkers_severity_map_file))
+        except ValueError:
+            LOG.warning('File not in JSON format: {}'.format(
+                self.checkers_severity_map_file))
```

Comment:
I think we should print here to the users that the severity levels will not be available if the file is missing or not in a valid json format.